conf/core/AttributeConflation.conf

```diff
@@ -16,6 +16,7 @@
   "poi.polygon.allow.cross.conflation.merging": "true",
   "poi.polygon.auto.merge.many.poi.to.one.poly.matches": "true",
   "reader.conflate.use.data.source.ids.1": "false",
+  "reader.conflate.use.data.source.ids.2": "true",
```

Comment:
For attribute conflation, secondary IDs are loaded, so that later they can be preserved.